utils/travis/publish-gh-pages.js

```diff
@@ -1,24 +1,15 @@
 #!/usr/bin/env node
 
-/* eslint-disable no-console */
-
-const ghPages = require('gh-pages');
-
 /**
- * This configuration will avoid logging the Github token if there is an error.
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
-ghPages.publish(
-  'demo',
-  {
-    repo: `https://${process.env.GITHUB_TOKEN}@github.com/zendeskgarden/react-components.git`,
-    silent: true
-  },
-  error => {
-    if (error) {
-      console.log(error);
-      throw error;
-    }
 
-    console.log('GH-Pages upload complete.');
-  }
-);
+const ghPages = require('gh-pages');
+
+ghPages.publish('demo', {
```

Comment:
I know we originally decided to remove the `add: true` option for ghPages, but I think we forgot about the "pre-publish" steps we do sometime to shared changes.

Without the `add` option those pre-published files will be wiped out. I think leaving those files around will be helpful.

Reply:
Ok, let's leave that as a conversation following this PR.